data/categories.txt

```diff
@@ -1249,7 +1249,7 @@ amenity-place_of_worship
 en:4place of worship|3temple|sights|U+1F64F
 ru:3храм|достопримечательность
 uk:3храм|пам’ятка|алтарь|мавзолей|пам’ятні місця
-de:3Anbetungsstätte|Tempel|Sehenswürdigkeit
+de:3Anbetungsstätte|Kirche|Moschee|Gotteshaus|Sehenswürdigkeit
```

Comment:
Alex Zolotarev schrieb:

> Kirche and Moschee should not be here, they have separate subtypes
> now. Is Tempel a bad word for "generic" place of worship of any
> religion? Is Gotteshaus a generic religion word?

Gotteshaus means "House of god", used in some regions of Germany for
Christian place of worships (a synonym for Kirche (church)). "Tempel" is
generic for places of worship not being Christian nor Muslim.
One may also add "Sakralbau" which refers to buildings for religious
purposes though for this I doubt that the majority of Germans would
search for "Sakralbau" when they want to see a Tempel, Kirche or
Moschee.